AppCenterDistribute/AppCenterDistribute/MSDistributePrivate.h

```diff
@@ -350,9 +339,11 @@ static NSString *const kMSTesterAppUpdateSetupFailedKey = @"MSTesterAppUpdateSet
 - (void)dismissEmbeddedSafari;
 
 /**
- * Start update workflow
+ * Start update workflow.
+ *
+ * @param onStart A flag that indicates whether it is called when Distribute is started or not.
  */
-- (void)startUpdate;
+- (void)startUpdateOnStart:(BOOL)onStart;
```

Comment:
We probably want to talk about this method signature even though this is internal use only.

Reply:
If the `onStart` parameter means what I think it means, the signature should just be `- (void)startUpdate:(BOOL)onStart`...

But `onStart` is supposed to indicate if the call is happening when the SDK is starting, right?  I think something like `isSdkStarting` would be clearer

Reply:
`updateShouldBeginOnStart:(BOOL)shouldBegin` could work. I think it is fine as is.

Reply:
I don't have a strong preference, if everything else is good we should merge this to unblock wrapper SDKs

Reply:
This is still open and we will revisit this soon.